app/components/primer/spinner_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use Primer::SpinnerComponent to let users know that content is being loaded.
+  class SpinnerComponent < Primer::Component
+
+    #
+    # @example 48|Default
+    #   <%= render(Primer::SpinnerComponent.new) %>
+    #
+    # @example 32|Small
+    #   <%= render(Primer::SpinnerComponent.new(size: 16)) %>
+    #
+    # @param size [Integer] Sets the size of the spinner
+    def initialize(size: 32, **kwargs)
```

Comment:
Do we want to standardize the `size` value? I could see us matching `OcticonComponent`, which has `small`, `medium`, and `large`. (16, 32, and 64)

Reply:
Good idea! Should we restrict the size to small/medium/large or should we allow a number to be passed too?

Reply:
I think OcticonComponent allows you to pass in a number too, so maybe we should be consistent with that?

Reply:
How about we start with a fixed list? We can always relax the API to accept numbers.

Reply:
Works for me 👍

Reply:
Updated ✅ 

Is there a way to write the comments so this arguments table is more helpful? 
![image](https://user-images.githubusercontent.com/4608155/96304759-8599a880-0fb1-11eb-857b-0c53c15051e5.png)

Reply:
@colebemis What might a more helpful table look like? 

If you're looking to list the possible values, this might be a good reference:

https://github.com/primer/view_components/blob/main/app/components/primer/blankslate_component.rb#L59

Perhaps we could write a helper for this?

Reply:
Exactly what I was looking for. Thanks! A helper could be nice but I'm not sure what we'd name it.